lib/parser.js

```diff
@@ -783,19 +783,27 @@ export default function Parser(options) {
           continue;
         }
 
-        j = xml.indexOf('>', i + 1);
-        if (j === -1) {
-          return handleError('unclosed tag');
-        }
+        for (x = i + 1; ; x++) {
+          const tmpw = xml.charCodeAt(x);
```

Comment:
Please drop TMP and use an existing temporary variable.

Reply:
@nikku 
Thank you for pointing this out.
I thought about using the variable w, it worked in the part on [L786](https://github.com/matsumotosyu/saxen/blob/57cd9373d27c65509ca3110ae5069c609ceacf41/lib/parser.js#L786), but seemed to have a side effect in the part on [L837](https://github.com/matsumotosyu/saxen/blob/57cd9373d27c65509ca3110ae5069c609ceacf41/lib/parser.js#L837) (changing the value of the w to refer to in subsequent processing).
(You didn't point out [L837](https://github.com/matsumotosyu/saxen/blob/57cd9373d27c65509ca3110ae5069c609ceacf41/lib/parser.js#L837), so I'm sorry if I'm over thinking it.)
Therefore, I have defined a new variable, v. (I thought it would be better to do the same for [L786](https://github.com/matsumotosyu/saxen/blob/57cd9373d27c65509ca3110ae5069c609ceacf41/lib/parser.js#L786), so I aligned the two.)
Is it okay this way?